bin/varnishd/cache/cache_rfc2616.c

```diff
@@ -220,6 +220,17 @@ RFC2616_Ttl(struct busyobj *bo, vtim_real now, vtim_real *t_origin,
 		*grace = rfc2616_time(p);
 	}
 
+	/*
+	 * The "must-revalidate" directive prohibits 
+	 * the cache from serving stale data,
+	 * according to RFC7234, 5.2.2.1.
+	 * This sets the grace to zero.
+	 */
+	if (*ttl >= 0 && http_GetHdrField(hp, H_Cache_Control,
+	    "must-revalidate", &p)) { // rfc7234,l,1303,1305
```

Comment:
@Dridi Here's where I refer to the (currently) non-existent page for RFC7234. Lines 1303-1305 talk about the staleness aspect. Hope the page magically appears once this comment is processed.